server/http_handler.go

```diff
@@ -223,6 +223,17 @@ func sessionHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	var res db.Sessions
+
+	if r.Method == "DELETE" {
+		vars := mux.Vars(r)
+		id := vars["id"]
+
+		if txn := dbserver.Instance.Delete(&res, id); txn.Error != nil {
+			jsonError(w, http.StatusInternalServerError, txn.Error)
```

Comment:
Eher BadRequest, also http 400?

Reply:
Kann ich gerne so machen. Bei sowas tue ich mich immer schwer. Wenn die Datenbank da zicken machen sollte, wäre es ja doch eher ein Interner Fehler auf dem Server, oder? Die Philosophie der IT 😁